alica_engine/include/engine/AlicaClock.h

```diff
@@ -128,10 +129,12 @@ class AlicaTime
 class AlicaClock
 {
 public:
-    AlicaClock() {}
-    virtual ~AlicaClock() {}
+    AlicaClock(AlicaEngine* ae)
```

Comment:
You still have this AlicaEngine reference. AlicaClock doesn't need it. Also remove it from other PR for AlicaRosClock.